repose-aggregator/src/docs/asciidoc/filters/attribute-mapping-policy-validation.adoc

```diff
@@ -0,0 +1,56 @@
+= Attribute Mapping Policy Validation Filter
+
+This filter will validate attribute mapping policies in requests to the origin service.
+
+The origin service is expected to be an Identity service.
+This filter will only take action if the HTTP request method is `PUT`.
+
+== General filter information
+* *Name:* attribute-mapping-policy-validation
+* *Released:* TBD
+* *Bundle:* repose-extensions-filter-bundle
+
+[NOTE]
+====
+This filter has no configuration.
+====
+
+== Prerequisites & Postconditions
+=== Required headers
+* `Content-Type` - Declares the content type of the policy (e.g., XML or JSON) which will be used during validation.
+
+=== Required preceding filters
+None.
+
+=== Request headers created
+None.
+
+=== Request body changes
+None.
+
+=== Recommended follow-on (succeeding) filters
+* <<saml-policy.adoc#,SAML Policy Translation Filter>> - These two filters work well together to provide mechanisms for attribute mapping policy management and application.
+
+=== Response body changes
+None.
+
+=== Response headers created
+None.
+
+=== Response Status Codes
+[cols="2", options="header,autowidth"]
+.Status Codes
+|===
+| Status Code
+| Reasons
+
+| `400`
```

Comment:
I don't particularly care either way. If we go with @wdschei's approach, I would propose the `400 Bad Request` format. If we go with @Mario-Lopez's approach, I assume we would link to RFC2616 (and the update status code RFC) like `[400](https://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html#sec10.4.1)`? So... everyone fight, the winner decides!

However, if we are going to add more static information to the status code, I would want a way to do so automatically (like as a pre-processing stage to building the docs). I'm not sure that's feasible, though.